packages/tsconfig/base.json

```diff
@@ -1,21 +1,25 @@
 {
   "$schema": "https://json.schemastore.org/tsconfig",
   "display": "Default",
-  "include": ["src"],
-  "exclude": ["dist", "build", "node_modules"],
+  "include": ["${configDir}/src"],
+  "exclude": [
+    "${configDir}/dist",
+    "${configDir}/build",
+    "${configDir}/node_modules"
+  ],
   "compilerOptions": {
     "composite": false,
-    "target": "ESNext",
-    "module": "ESNext",
-    "moduleResolution": "bundler",
```

Comment:
Why this? You wrote `"moduleResolution": "bundler"` in all other configs, why just not having it here instead?

Reply:
I assumed that they were all different (not all of them are bundled), but everything else (i.e., `Node16`) breaks. I can recentralize it if desired.

Reply:
yes please